core/controllers/collection_editor.py

```diff
@@ -122,15 +122,14 @@ def get(self, collection_id):
                 rights_manager.ACTIVITY_TYPE_COLLECTION, collection_id))
 
         self.values.update({
-            'is_private': rights_manager.is_collection_private(collection_id),
             'can_edit': can_edit,
-            'collection_id': collection.id,
-            'title': collection.title,
             'can_unpublish': rights_manager.Actor(
                 self.user_id).can_unpublish(
                     rights_manager.ACTIVITY_TYPE_COLLECTION, collection_id),
-            'SHOW_COLLECTION_NAVIGATION_TAB_SETTINGS': (
-                feconf.SHOW_COLLECTION_NAVIGATION_TAB_SETTINGS),
+            'collection_id': collection.id,
+            'is_private': rights_manager.is_collection_private(collection_id),
+            'nav_mode': feconf.NAV_MODE_CREATE,
```

Comment:
This is for specializing the navbar? I worry that this is ambiguous with the exploration editor, but I suppose it's fine for now.